.env.template

```diff
@@ -10,6 +10,10 @@ PLAID_SECRET_SANDBOX=
 
 PLAID_ENV=sandbox
 
+# If you want to test the app using a payment processor, set PROCESSOR to 'true'
```

Comment:
do we know which payment processor this would cause the user to use?

Reply:
I will put in a note that it is Dwolla for the sample app